packages/sdk-js/test/cases.json

```diff
@@ -1,5 +1,5 @@
 {
-  "specVersion": "0.4.2",
+  "specVersion": "0.5.0",
```

Comment:
When we bump the spec, we should update the `build-your-own` page to match.  There are 2 sections that need to be updated:
1. At the top where it shows the latest spec version
2. Update the changelog at the bottom